tests/rbthemis/ssession_test.rb

```diff
@@ -93,4 +94,34 @@ def test_session
       Thread.new { client }
     ].each(&:join)
   end
+
+  def test_invalid_key_public_ec
+    begin
+      session = Themis::Ssession.new(
+        'client', @client_pub_key, CallbacksForThemis.new(@pub_keys))
+    rescue
+    else
+      assert(false)
+    end
+  end
```

Comment:
Shouldn't those tests check that `Ssession` throws a particular error we expect? :thinking:

Reply:
I'm not an expert in Ruby, my attempts to make it only catch `ThemisError` didn't succeed so far. BTW, the only possible exception we can catch is the `ThemisError`. However, it can contain different string error messages.

Reply:
It looks something like this:

```ruby
  def test_invalid_key_public_ec
    assert_raise(Themis::ThemisError) do
      session = Themis::Ssession.new(
        'client', @client_pub_key, CallbacksForThemis.new(@pub_keys))
    end
  end
```